### PR TITLE
Create configuration/environment-variable

### PR DESCRIPTION
**Pull request by 6293**

### CODE REVIEW

File: directory.json [hunk 1]
```diff
@@ -295,7 +295,16 @@
     },
     {
       "title": "参数配置",
-      "path": "configuration/configuration"
+      "children": [
+        {
+          "title": "参数配置 list",
```
Comment (6293) [inline directory.json]: help to fill in proper 中文

File: directory.json [hunk 2]
```diff
@@ -295,7 +295,16 @@
...
+          "path": "configuration/configuration"
+        },
+        {
+          "title": "环境变数",
```
Comment (6293) [inline directory.json]: does this make sense?

Comment (zmstone) [reply directory.json]: 环境变量

File: directory.json (hunk 1 above)
Comment (zmstone) [reply directory.json]: 参数配置列表

File: en_US/configuration/environment-variable.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+# Configuration from environment variable
+
+Use environment variables to set configuration values temporarily.
+
+By default, EMQ X maps environment variables with prefix ``EMQX_`` to key-value pairs in configuration files.
+
+- Default mapping rules:
+
+  + Prefix ``EMQX_`` is removed
+  + All upper case letters are replaced with lower case letters
```
Comment (zmstone) [inline en_US/configuration/environment-variable.md]:
```suggestion
  + Upper case letters are mapped to lower case
```

File: en_US/configuration/environment-variable.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+  + ``__`` is replaced with ``.``
```
Comment (zmstone) [inline en_US/configuration/environment-variable.md]:
```suggestion
  + Double underscore``__`` is mapped to ``.``
```

**Comment by 6293** [comment]
@zmstone thanks, changes applied. Now I need someone to help writing Chinese version ( that is why CI fails ).

File: en_US/configuration/environment-variable.md [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+
+Use environment variables to set configuration values temporarily.
+
+By default, EMQ X maps environment variables with prefix ``EMQX_`` to key-value pairs in configuration files.
+
+- Default mapping rules:
+
+  + Prefix ``EMQX_`` is removed
+  + Upper case letters are mapped to lower case letters
+  + Double underscore``__`` is mapped to ``.``
```
Comment (zmstone) [inline en_US/configuration/environment-variable.md]: nit, there is a space missing
```suggestion
  + Double underscore ``__`` is mapped to ``.``
```

Comment (6293) [reply en_US/configuration/environment-variable.md]: ty fixed

**Comment by zmstone** [comment]
builds are failing.

**Comment by 6293** [comment]
@zmstone Not a build failure. I feel CI is broken because it fails even when no changes exist on markdown.

https://github.com/emqx/emqx-docs/pull/665

File: en_US/configuration/environment-variable.md [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+
+...
+
+Starting emqx on node emqx@127.0.0.1
+Start http:management listener on 9000 successfully.
+```
+
+::: tip Tip
+环境变量只在运行 EMQ X 的终端中有效，且没有持久化,
+在一个终端中设置的环境变量无法在另一个终端中使用。
```
Comment (6293) [inline en_US/configuration/environment-variable.md]: you wrote Chinese in English version

**Comment by zmstone** [comment]
related: https://github.com/emqx/emqx/pull/4267

**Review by zmstone — APPROVED** [review]

**Pull request merged**